pkg/reconciler/pipelinerun/pipelinerun_test.go

```diff
@@ -154,29 +154,30 @@ func initializePipelineRunControllerAssets(t *testing.T, d test.Data, opts pipel
 	}, cancel
 }
 
-// getTaskRunCreations will look through a set of actions to find all task run creation actions and return the set of
-// them. It will fatal the test if none are found, or if less actions were found than expected.
-func getTaskRunCreations(t *testing.T, actions []ktesting.Action, minActionCount int) []*v1beta1.TaskRun {
+// getTaskRunCreations will look through a set of taskruns generated by a specific pipelinerun and return the set of
+// them. It will fatal the test if none are found or an error occurred
+func getTaskRunCreations(ctx context.Context, t *testing.T, clients test.Clients, namespace string, prName string) []*v1beta1.TaskRun {
```

Comment:
I'd suggest renaming this function and updating its docstring, since it now checks the number of taskruns associated with a pipelinerun, not the number of "creations" (i.e. it could be used for testing updates as well).

In addition, this used to check the expected number of TaskRun creations, and no longer does. I think it would make sense for this function to accept an additional parameter for the expected number of TaskRuns associated with the PipelineRun, and verify the number of TaskRuns is what is expected.